pulp_smash/tests/pulp2/docker/utils.py

```diff
@@ -57,3 +59,11 @@ def write_manifest_list(cfg, manifest_list):
         )
     )
     return file_path, dir_path
+
+
+skip_if = partial(selectors.skip_if, exc=SkipTest)  # pylint:disable=invalid-name
```

Comment:
Any special reason for repeating the definition of this `partial` multiple times instead of having a single `selectors.unittest_skip_if` and reuse it?

Reply:
I'd like to make sure that Pulp Smash can be used with many different test runners. One way of advancing that goal is to make sure that the tools it provides reference unittest as little as possible.